PaddleCV/object_detection/tools/train.py

```diff
@@ -40,33 +40,53 @@
 
 
 def main():
-    args = parse_args()
+    parser = ArgsParser()
+    parser.add_argument(
+        "-r",
+        "--resume_checkpoint",
+        default=None,
+        type=str,
+        help="Checkpoint path for resuming training.")
+    parser.add_argument(
+        "--eval",
+        action='store_true',
+        default=False,
+        help="Whether to perform evaluation in train")
+    parser.add_argument(
+        "-s",
+        "--savefile",
+        default=None,
+        type=str,
+        help="Evaluation file name, default to bbox.json and mask.json."
```

Comment:
This arg is not needed in train and eval? only need in infer.

Reply:
it is used by `eval_results`